account_due_list_payment_mode/views/payment_view.xml

```diff
@@ -1,31 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
+<odoo>
 
-        <record id="view_payments_mode_tree" model="ir.ui.view">
-            <field name="name">Payments MOde</field>
-            <field name="model">account.move.line</field>
-            <field name="inherit_id" ref="account_due_list.view_payments_tree"/>
-            <field name="arch" type="xml">
-                <field name="partner_ref" readonly="1" position="after">
-                    <field name="payment_mode_id" readonly="1"/>
-                </field>
+    <record id="view_payments_mode_tree" model="ir.ui.view">
+        <field name="name">Payments Mode</field>
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="account_due_list.view_payments_tree"/>
+        <field name="arch" type="xml">
+            <field name="partner_ref" position="after">
+                <field name="payment_mode_id" readonly="1"/>
             </field>
-        </record>
+        </field>
+    </record>
 
-        <record id="view_payments_mode_filter" model="ir.ui.view">
-            <field name="name">Payments Mode Select</field>
-            <field name="model">account.move.line</field>
-            <field name="inherit_id" ref="account_due_list.view_payments_filter"/>
-            <field name="arch" type="xml">
-                <field name="date_maturity" position="after">
-                    <field name="payment_mode_id"/>
-                </field>
-                <filter string="Month" icon="terp-go-month" domain="[]" context="{'group_by':'date_maturity'}" position="after">
-                    <filter string="Payment Mode" icon="terp-dolar" domain="[]" context="{'group_by':'payment_mode_id'}"/>
-                </filter>
+    <record id="view_payments_mode_filter" model="ir.ui.view">
+        <field name="name">Payments Mode Select</field>
```

Comment:
@pedrobaeza You are sayng to don't use an explict name just a namespace like:

https://github.com/OCA/account-analytic/blob/10.0/account_analytic_no_lines/views/account_analytic_account.xml#L8

```
<record model="ir.ui.view" id="account_analytic_account_form_view">
        <field name="name">account.analytic.account.form (in account_analytic_no_lines)</field>
        <field name="model">account.analytic.account</field>
        <field name="inherit_id" ref="analytic.view_account_analytic_account_form"/>
        <field name="arch" type="xml">
            <xpath expr="//div[@name='button_box']/button[@name=%(analytic.account_analytic_line_action)d]" position="attributes">
                <attribute name="groups">account_analytic_no_lines.hide_analytic_entries</attribute>
            </xpath>
        </field>
    </record>
```

In this case
```
<field name="name">account.move.select (in account_due_list_payment_mode)</field>
```

cc: @vonpupp